test/unit/test-friendly-iframe-embed.js

```diff
@@ -669,17 +669,17 @@ describes.realWin('friendly-iframe-embed', {amp: true}, (env) => {
   it('should call for remeasure upon resize', async () => {
     const iframe = document.createElement('iframe');
     const {promise, resolve} = new Deferred();
-    const mutateSpy = env.sandbox.stub(
-      Services.mutatorForDoc(env.ampdoc),
-      'mutateElement'
-    );
 
     await installFriendlyIframeEmbed(iframe, document.body, {
       url: 'https://acme.org/url1',
       html: '<a id="a1"></a>',
       extensions: [],
     });
 
+    const mutateSpy = env.sandbox.stub(
+      Services.mutatorForDoc(env.ampdoc),
+      'mutateElement'
+    );
```

Comment:
Why does this fix it?

Reply:
The browser somehow triggers a resize event after iframe is being created, therefore causes a `mutateElement` call (https://github.com/ampproject/amphtml/blob/main/src/friendly-iframe-embed.js#L412). The event emits probabilistically.

Moving spy to after iframe being created will prevent browser resize event from affecting the test.

Reply:
Thanks, can you add this to PR description?